mem0/client/main.py

```diff
@@ -91,7 +93,7 @@ def __init__(
 
         if organization or project:
             warnings.warn(
-                "Using 'organization' and 'project' parameters is deprecated and will be removed in version 0.1.40. "
+                "Using 'organization' and 'project' parameters is deprecated and will be removed in version 0.1.60. "
```

Comment:
Let's remove support for this since we have already passed the 0.1.40 version long back.